perfkitbenchmarker/providers/aws/aws_emr.py

```diff
@@ -98,22 +117,45 @@ def _Exists(self):
     else:
       return True
 
-  def _WaitUntilReady(self):
+  def _IsReady(self):
     """Check to see if the cluster is ready."""
     cmd = self.cmd_prefix + ['describe-cluster', '--cluster-id',
                              self.cluster_id]
-    tries = 0
-    while tries < READY_CHECK_TRIES:
-      tries += 1
-      stdout, _, rc = vm_util.IssueCommand(cmd)
-      result = json.loads(stdout)
-      if result['Cluster']['Status']['State'] == READY_STATE:
-        return True
-      time.sleep(READY_CHECK_SLEEP)
-    return False
+    stdout, _, rc = vm_util.IssueCommand(cmd)
+    result = json.loads(stdout)
+    if result['Cluster']['Status']['State'] == READY_STATE:
+      return True
+    else:
+      return False
+
+  def _GetLogBase(self):
+    """Gets the base uri for the logs."""
+    cmd = self.cmd_prefix + ['describe-cluster', '--cluster-id',
+                             self.cluster_id]
+    stdout, _, _ = vm_util.IssueCommand(cmd)
+    result = json.loads(stdout)
+    if 'LogUri' in result['Cluster']:
+      self.logging_enabled = True
+      log_uri = result['Cluster']['LogUri']
+      if log_uri.startswith('s3n'):
+        log_uri = 's3' + log_uri[3:]
+      return log_uri
+    else:
+      return None
+
+  @vm_util.Retry()
```

Comment:
If you aren't going to specify parameters to Retry you might as well just use vm_util.IssueRetryableCommand